controller/src/beerocks/bml/bml.h

```diff
@@ -151,6 +151,46 @@ int bml_event_register_cb(BML_CTX ctx, BML_EVENT_CB cb);
 int bml_set_wifi_credentials(BML_CTX ctx, const char *ssid, const char *pass, int sec, int vap_id,
                              int force);
 
+/**
+ * Sets the Wi-Fi credentials for the client with AL-MAC.
+ *
+ * @param [in] ctx BML Context.
+ * @param [in] al_mac client AL-MAC address 
+ * @param [in] ssid The SSID for the network.
+ * @param [in] network_key The passphrase for the network.
+ * @param [in] radio The type of the radio 2.4g or 5g
+ * @param [in] bss_type The type of bss, can be fronthaul, backhaul, fronthaul-backhaul
+ *
+ * @return BML_RET_OK on success.
+ */
+int bml_set_wifi_credentials_new(BML_CTX ctx,const char* al_mac, const char* ssid, const char* network_key,
+                                 const char* radio, const char* bss_type);
+
+/**
+ * Removes the Wi-Fi credentials for the client with AL-MAC
+ *
+ * @param [in] ctx BML Context.
+ * @param [in] al_mac client AL-MAC address 
+ *
+ * @return BML_RET_OK on success.
+ */
+int bml_clear_wifi_credentials(BML_CTX ctx, const char* al_mac);
+
+/**
+ * Updates the Wi-Fi credentials for the client with AL-MAC and sends AP Renew request
+ *
+ * @param [in] ctx BML Context.
+ * @param [in] al_mac client AL-MAC address 
+ * @param [in] ssid The SSID for the network.
+ * @param [in] network_key The passphrase for the network.
+ * @param [in] radio The type of the radio 2.4g or 5g
+ * @param [in] bss_type The type of bss, can be fronthaul, backhaul, fronthaul-backhaul
+ *
+ * @return BML_RET_OK on success.
+ */
+int bml_update_wifi_credentials(BML_CTX ctx,const char* al_mac, const char* ssid, const char* network_key,
```

Comment:
"update" just commits the credentials that were added before with set_wifi_credentials. So update takes only al_mac as argument.